src/org/traccar/web/WebServer.java

```diff
@@ -156,6 +161,19 @@ private void initOldWebApp() {
         handlers.addHandler(app);
     }
 
+    private void initMedia() {
+        ServletContextHandler servletHandler = new ServletContextHandler(ServletContextHandler.SESSIONS);
+        servletHandler.setContextPath("/api/media");
```

Comment:
We have to use sub path to make SessionManager use same sessions as for `/api`

Reply:
Can we add it to the API ServletContextHandler?